annotations/augmenter_test.go

```diff
@@ -113,11 +165,38 @@ func TestAugmentAnnotations(t *testing.T) {
 	conceptsSearchAPI.AssertExpectations(t)
 }
 
+
+func TestAugmentAnnotationsMultiPredicatePerConcept(t *testing.T) {
+	matcher := mock.MatchedBy(func(l1 []string) bool {
+		return assert.ElementsMatch(t, l1,testMultiPredicateConceptIDs)
```

Comment:
:+1: ... though you have to read to godoc to be certain the order of elements isn't significant.